base_layer/wallet_ffi/wallet.h

```diff
@@ -868,6 +870,8 @@ bool wallet_start_recovery(struct TariWallet *wallet, struct TariPublicKey *base
 /// None
 bool wallet_set_one_sided_payment_message(struct TariWallet *wallet, const char *message, int *error_out);
 
+struct TariFeePerGramStatsResponse *wallet_get_fee_estimates_per_block(struct TariWallet *wallet, unsigned int count, int *error_out);
```

Comment:
You'll need to add the remaining fee_per_gram functions to the header

Reply:
Ahhh, there is so much stuff that we could just generate ;-) but it's not worth of doing so...